src/features/user/ManageProjects/components/ProjectSites.tsx

```diff
@@ -237,7 +260,7 @@ export default function ProjectSites({
                       {...viewport}
                       center={[longitude, latitude]}
                       zoom={[zoom]}
-                      style="mapbox://styles/mapbox/streets-v11?optimize=true" // eslint-disable-line
+                      style={style} // eslint-disable-line
```

Comment:
We need to update the default style to include road level details.

Reply:
here I think we are using OSM street map, it should have road level data